test/library/standard/Socket/SKIPIF

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+has_libevent = subprocess.getoutput("ldconfig -v | grep libevent - | wc -l")
```

Comment:
Normally we use pkg-config for such things

Reply:
Also ldconfig -v and pkg-config won't exist on all systems, so you have to fail gracefully in that event

Reply:
See test/library/packages/Crypto.skipif for an example